tests/tests_insights_client_register.yml

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Basic insights-client register/unregister
+  hosts: all
+  gather_facts: true
+
+  tasks:
+    - name: Unregister insights
+      include_role:
+        name: linux-system-roles.rhc
+      vars:
+        rhc_insights:
+          state: absent
```

Comment:
in addition to this, we also need
```yaml
  rhc_state: absent
```
otherwise it will try to register to rhsm (and it fails, because `rhc_auth` was not specified)

Reply:
`insights-client` needs that rhsm is registered to run. I don't think this is a good option to test the `insights` side.

Reply:
right :) however this tasks is trying to unregister from insights, and not setting `rhc_state` means it will try to register with rhsm (and fail because of lack of credentials)

Reply:
Yes, you are right. I already change it in the last pushed changes